flash.bat

```diff
@@ -0,0 +1 @@
+idf.py -p COM5 -b 2000000 build flash monitor
```

Comment:
I'm all for helper batch files with `%UserProfile%` and such in them, but `COM5` is specific to _your_ machine. I would much rather that be an environment variable too, like `%SWADGE_COM_PORT%` or something

Reply:
I can implement that